### PR TITLE
add markdown linting

### PR DESCRIPTION
**Pull request by jelhan**
Setup and configuration are based on [ember-learn/guides-source](https://github.com/ember-learn/guides-source).

This is kind of preparation work for #5. As we add more and more content we should make sure that we follow shared conventions.

### CODE REVIEW

File: README.md
Comment (knownasilya) [inline README.md]: Unfortunate that this removed the `alt`

File: README.md
Comment (knownasilya) [inline README.md]: Can probably remove 'Easy-peasy'. It seems unnecessary.

**Review by knownasilya — APPROVED** [review]
Looks good generally. I left a couple comments, but nothing blocking from merging

File: README.md
Comment (MelSumner) [inline README.md]:
```suggestion
Once we've had a chat, we can add you as a contributor.
```

File: README.md
Comment (MelSumner) [reply README.md]: Let's not do it if it removes the `alt`.

**Review by MelSumner — CHANGES_REQUESTED** [review]
A few minor changes. Otherwise looks good! Thank you for working on this! 👍

File: README.md
Comment (jelhan) [reply README.md]: I have double-checked that it still renders the `alt` attribute with `"Deploys by Netlify"`. The rendered HTML markup is:

```html
<p><a href="https://www.netlify.com" rel="nofollow">
<img src="https://camo.githubusercontent.com/89bdfa968b425fbf4502e6a0ccd2567501dd82bf/68747470733a2f2f7777772e6e65746c6966792e636f6d2f696d672f676c6f62616c2f6261646765732f6e65746c6966792d636f6c6f722d616363656e742e737667" alt="Deploys by Netlify" data-canonical-src="https://www.netlify.com/img/global/badges/netlify-color-accent.svg" style="max-width:100%;">
</a></p>
```

The `alt` attribute is set by the content in square brackets. The image syntax in markdown is: `alt text`.

**Comment by jelhan** [comment]
Thanks a lot for your reviews. Have addressed all comments.

Just noticed that a `.editorconfig` was missing in this repository. I added it in https://github.com/adopted-ember-addons/program-guidelines/pull/7/commits/6d65a74de833d501cbb440ae4490fd0fb7a2a60b. It's based on the one generated by `ember new`.

**Comment by jelhan** [comment]
What is needed to get this merged? I don't have merge permissions myself in this repository.

**Comment by knownasilya** [comment]
I can merge, but would want @MelSumner to approve since she requested changes.

**Comment by jelhan** [comment]
@knownasilya Thanks for your quick response!

Rebased this merge request to resolve conflicts after #9 had been merged.

**Review by MelSumner — APPROVED** [review]
Thank you for doing this! 👍

**Pull request merged**
